template-helpers/homepage.js

```diff
@@ -28,6 +32,11 @@ function makeLanding(args) {
     });
   }
 
+  // Growth Experiment
+  if (featuredBreach && experimentBranchB) {
+    landingCopy.subhead = "Your free Firefox account alerts you if you’ve been involved in a known data breach.";
```

Comment:
Just verifying - do we want this to be English only?

Reply:
@lesleyjanenorton @groovecoder Correct – it's being filtered in this argument: 
https://github.com/mozilla/blurts-server/blob/1638-homepage-fxa-experiment/controllers/home.js#L31

We're using a direct string like this in order to not deploy any l10n resources.